src/style.css

```diff
@@ -0,0 +1,18 @@
+body {
+	margin: 0px;
+}
+
+div {
```

Comment:
do nott add styles to tags, it is too common. Imagine you need to add other div to page, with different styles